semisupervised/StackedAutoEncoderSSL.py

```diff
@@ -148,12 +151,12 @@ def fit(self, X, y, is_pretrain=True, validation_data=None):
                     min_loss = valid_loss
                 else:
                     patience_counter += 1
-                if ep % self.verbose == 0:
+                if self.verbose:
```

Comment:
Verbose is int type. If the epoch is super large, it will show the information in each `verbose` epoch.

Reply:
Posted in a separate comment but essentially this leads to a div by 0 error on verbose=0, I'd suggest:

`if self.verbose and (ep % self.verbose == 0):`